products/compute/api.yaml

```diff
@@ -1088,6 +1088,51 @@ objects:
         description: |
           A reference to the region where the regional address resides.
         output: true
+      - !ruby/object:Api::Type::Integer
+        name: 'prefixLength'
+        description: |
+          The prefix length of the IP range. If not present, it means the
+          address field is a single IP address.
+          
+          This field is not applicable to addresses with addressType=EXTERNAL.
+        min_version: beta
+      - !ruby/object:Api::Type::Enum
+        name: 'addressType'
+        description: |
+          The type of the address to reserve, default is EXTERNAL.
+
+          * EXTERNAL indicates public/external single IP address.
+          * INTERNAL indicates internal IP ranges belonging to some network.
+        values:
+        - :EXTERNAL
+        - :INTERNAL
+        default_value: :EXTERNAL
+        min_version: beta
+      - !ruby/object:Api::Type::Enum
+        name: 'purpose'
+        description: |
+          The purpose of the resource. For global internal addresses it can be
+          
+          * VPC_PEERING - for peer networks
+          
+          For regional internal addresses, it can be
```

Comment:
Since this is the GlobalAddress resource, I don't think this value is worth documenting.

Since then it seems like there's only one value this field could take, I'm curious if it's something that we should actually give users the option to set. I assume there's a difference between setting it to VPC_PEERING and just not setting it at all?